File: C Programming/problem on numbericals/problem4.c
/*
4. Accept one number and check whether is is divisible by 5 or not
	
	Author: Milind Nalawade
	
	Date:18/06/2021
	
	language : C Programming
*/

#include<stdio.h>


// Entry point function

int main()
{
	int divident=0,divisor=0;
	printf("Enter the divident : ");
	scanf("%d",&divident);
	printf("Enter the divisor : ");
	scanf("%d",&divisor);
	
	if(divident%divisor==0)
	{
		printf("%d is divisible by %d",divident,divisor);
	}
	else
	{
		printf("%d is not divisible by %d",divident,divisor);
	}
}

File: Python Programming/problems on numbericals/problem4.py
'''
4. Accept one number and check whether is is divisible by 5 or not
	
	Author: Milind Nalawade
	
	Date:18/06/2021
	
	language : python3
	
'''
#===============
#Entry point function
#===============

def main():
	
	no1=int(input("Enter Divident : "))
	no2=int(input("Enter divisor : "))
	
	if no1%no2==0:
		print(" {} is divisible by {}".format(no1,no2))
	else:
		print(" {} is not divisible by {}".format(no1,no2))
		
#============
#starter
#============
if __name__=="__main__":
	main()